card/mod/05_email/set/self/edited.rb

```diff
@@ -11,7 +11,7 @@ def title
 end
 
 def label
-  "follow what I've edited"
+  "follow if I edited"
 end
```

Comment:
changed my mind about the wording after looking at it a bit.  may change again after feedback :)